test/unit/helpers/dom/EnrollChoicesForm.js

```diff
@@ -1,4 +1,4 @@
-define(['jquery', './Form'], function ($, Form) {
+define(['jquery', './Form', 'underscore'], function ($, Form, _) {
```

Comment:
nit: missed this, but generally we have jquery and underscore before other files
```
define(['jquery', 'underscore', './Form'], function ($, _, Form) {
```